doc/TechnicalFAQs.md

```diff
@@ -0,0 +1,128 @@
+
+## Installation Failure caused by ClickOnce Cache<a name="clickonce-cache"></a>
+#### Error Message
+![Alt text](../doc/images/TechnicalFAQ/clickonce-cache.png)
+#### Solution
+1. Follow these [instructions](https://msdn.microsoft.com/en-sg/library/ms178739.aspx) to install Visual Studio Tools for Office runtime
+1. Open `Start`
+1. Type `cmd` and press `Enter`
+1. Paste the following command and press `Enter`: `rundll32 %windir%\system32\dfshim.dll CleanOnlineAppCache`
+1. Reinstall PowerPointLabs again
+
+## Referenced Assembly Not Installed<a name="assembly-not-installed"></a>
+#### Error Message
+![Alt text](../doc/images/TechnicalFAQ/assembly-not-installed.png)
+```
+System.Runtime.InteropServices.COMException (0x800736B3): The referenced assembly is not installed on your system. (Exception from HRESULT: 0x800736B3)
+ at Microsoft.VisualStudio.Tools.Applications.Deployment.ClickOnceAddInDeploymentManager.DownloadAddIn(TimeSpan timeout)
+ at Microsoft.VisualStudio.Tools.Applications.Deployment.ClickOnceAddInDeploymentManager.InstallAddIn()
+```
+#### Solution
+1. Open `Start`
+1. Type `cmd` and press `Enter`
+1. Paste the following command and press `Enter`: `rundll32 %windir%\system32\dfshim.dll CleanOnlineAppCache`
+1. Reinstall PowerPointLabs
+1. Else, delete folder `%LocalAppData%\Apps\2.0` and reinstall PowerPointLabs again
+
+## Corrupted Zip Installer<a name="corrupted-zip"></a>
+#### Error
+Unable to unzip the installer and/or error message associated with operation of unzipping the installer
+#### Solution
+1. Re-download the installer files and try installing again
+1. If multiple re-downloads fail, download the setup.exe directly [here](https://www.google.com/url?q=http%3A%2F%2Fwww.comp.nus.edu.sg%2F~pptlabs%2Fdownload-78563%2FPowerPointLabsInstaller.zip) and re-install
+
+## Installation Error due to Preexisting Similar Application<a name="preexisting-application"></a>
+#### Error Message
+![Alt text](../doc/images/TechnicalFAQ/preexisting-app.png)
+#### Solution
+1. Go to `File Explorer`
+1. Type in `%LocalAppData%\Apps\2.0` in the navigation bar and press `Enter`
+1. Using the search bar on the top-right corner, locate the `Manifests` folder
+1. Delete all files starting with `powe...` as shown in the below image
+![Alt text](../doc/images/TechnicalFAQ/preexisting-app2.png)
+
+## Error Loading Manifest from MahApps.Metro.dll<a name="error-manifest"></a>
+#### Error Message
+![Alt text](../doc/images/TechnicalFAQ/error-manifest.png)
+#### Solution
+1. Open `Start`
+1. Type `cmd` and press `Enter`
+1. Type in `cd C:\Users\pvolowz\AppData\Local\Temp\Deployment\1HKX7LKW.XZD\HYEJO9YP.GNM`
+1. Type in `tasklist.exe /m MahApps.Metro.dll`
+1. Determine which process is using the dll and end it if possible
+1. Else, delete folder `C:\Users\pvolowz\AppData\Local\Temp\Deployment` and reinstall PowerPointLabs again
```

Comment:
This too, the more general version is  cd %LocalAppData%\Temp\Deployment

However I think it'll be good to provide both the general version and the example

Reply:
Maybe could also add:

> Before deleting the folder, do also try temporarily disabling any anti-virus software running in the computer in case the anti-virus software is the one preventing the use of the .dll file.